### DISCUSSION TITLE
"Sending command" issue

### DISCUSSION THREAD

**Discussion by CatAnonymous** [discord-php/DiscordPHP#1133] [Q&A] [answered]
The text "Sending command" was not disabled after the response was sent, is referReply() supported like python?

```
    $discord->listenCommand('test', function (Interaction $interaction) {
        $content = $interaction->data->options->offsetGet('text')->value;

        $interaction->respondWithMessage(MessageBuilder::new()->setContent('Your text: ' . $content));

    });
```

**Comment by key2peace** [comment]
$interaction->acknowledgeWithResponse(); should trigger the change from sending to thinking,
then instead of respondWithMessage, use updateFollowUpMessage

**Reply by CatAnonymous** [reply, marked as answer]
> $interaction->acknowledgeWithResponse(); should trigger the change from sending to thinking,
> then instead of respondWithMessage, use updateFollowUpMessage

 i use updateOriginalResponse instead of updateFollowUpMessage, it worked. Thankyou
